t/modules/Tools/Compare.t

```diff
@@ -516,6 +516,118 @@ subtest number => sub {
 
 };
 
+subtest bool => sub {
+    my @true = (1, 'yup', '0 but true', ' ', {});
+    my @false = (0, '0', '', undef);
+
+    for my $true (@true) {
+        for my $true2 (@true) {
+            is($true2, bool($true), "Both true");
+
+            my $line = __LINE__ + 2;
+            is(
+                intercept { is($true2, bool($true, negate => 1)) },
+                array {
+                    fail_events Ok => {pass => 0};
+                    event Diag => sub {
+                        call message => table(
+                            header => [qw/GOT OP CHECK LNs/],
+                            rows   => [["TRUE ($true2)", 'XOR', "TRUE ($true)", $line]],
+                        );
+                    };
+                    end;
+                },
+                "true($true2) + true($true) + negate"
+            );
+        }
+
+        for my $false (@false) {
+            is($false, bool($true, negate => 1), "true + false + negate");
+
+            my $render = 'FALSE (' . (defined($false) ? length($false) ? $false : "''" : 'undef') . ')';
+
+            my $line = __LINE__ + 2;
+            is(
+                intercept { is($false, bool($true)) },
+                array {
+                    fail_events Ok => {pass => 0};
+                    event Diag => sub {
+                        call message => table(
+                            header => [qw/GOT OP CHECK LNs/],
+                            rows   => [[$render, 'XNOR', "TRUE ($true)", $line]],
```

Comment:
Unless the user has a very firm grounding in boolean logic, they're not going to know what XNOR is. Most folks even get confused by XOR.

Just use `==` and `!=`.